safe/definitions/minimum_needs.py

```diff
@@ -70,9 +68,9 @@ def _initializes_minimum_needs_fields():
             field_name = '{namespace}__{field_name}'.format(**format_args)
             # It's easier to debug if we have the real value from the
             # postprocessor. That's why I put a Double. ET.
-            field_type = QVariant.Double
+            field_type = qvariant_numbers
```

Comment:
Yes, @Gustry I think we should brought it up in the discussion. Like I said, it will bring number inconsistencies if it is summed, especially if the number is small.
The minimum needs and gender postprocessor is actually from a different cause.

- For the minimum needs it was caused by the change, because I made it as integers
- I suspect that it was always that case with the gender postprocessor (I didn't change that). It was one of the reason I propose to change displaced and postprocessors to decimals.

From this, we can either make all becomes integers and ignores small numbers discrepancies. Or all should be decimals, even if it doesn't make sense in the attribute table (like half people or half toilet 😄 ). CC @timlinux you might want to read this.